README.md

```diff
@@ -1,68 +1,66 @@
-# Oracle Connector
+# Oracle Database Connector
 
 This is an Arrow connector to Oracle Database.
 
-## Installation
+## Prerequisites
+
+This connector requires Oracle Instant Client.
```

Comment:
The Instant Client only applies to Mac OS. Since connectors can also be used on Windows/Linux please refer to them as Oracle client libraries.